### DISCUSSION TITLE
Collapsible show info on show detail page

### DISCUSSION THREAD

**Discussion by Rouzax** [pymedusa/Medusa#3663] [Ideas & Feature requests]
Would it be possible to make the show info on the show detail page collapsible?
Especially on a mobile device it takes up a lot of space.
![image](https://user-images.githubusercontent.com/4103090/35342302-056b2f3e-0128-11e8-95c9-413c795b4683.png)

**Comment by Rouzax** [comment]
@p0psicles: all the issues you've been closing are they still tracked somewhere or dropped completely?

**Comment by p0psicles** [comment]
Their closed not gone. So anyone can search and find it. It's just as visible as when hidden between 3000 open issues

**Comment by Rouzax** [comment]
I understand.
Does that mean they are still considered feature requests?

**Comment by p0psicles** [comment]
Anyone can do feature requests and anyone can do pull requests. It's open source.

**Comment by Rouzax** [comment]
@medariox: are you reopening all requests again?

**Comment by medariox** [comment]
@Rouzax 
No, but you can bump the ones you are interested in.
